mis_builder_budget/security/mis_budget.xml

```diff
@@ -24,4 +24,11 @@
         <field name="perm_unlink" eval="1"/>
     </record>
 
+    <record id="mis_budget_multi_company_rule" model="ir.rule">
+        <field name="name">mis.budget multi company</field>
+        <field name="model_id" ref="model_mis_budget"/>
+        <field name="global" eval="True"/>
```

Comment:
This one is computed

Reply:
@adrienpeiffer what do you mean?

Reply:
The ```global``` field of ```ir.rule``` is computed. So, you shouldn't set this one.